src/modules/licence-import/mappers/agreement.js

```diff
@@ -0,0 +1,35 @@
+const helpers = require('@envage/water-abstraction-helpers');
+const date = require('./date');
+const { groupBy, sortBy, flatMap } = require('lodash');
+
+const LICENCE_LEVEL_AGREEMENT_TYPES = ['S127', 'S130S', 'S130T', 'S130U', 'S130W'];
```

Comment:
Could this filtering be added to the SQL instead to have less code here that is not tested either? Also means less data crunching required in this part.